### DISCUSSION TITLE
Can't get wfreerdp /monitors argument to work

### DISCUSSION THREAD

**Discussion by espenu** [FreeRDP/FreeRDP#9532] [Q&A] [answered]
We have classroom setup with Win10 (21H2) client PCs connecting to Win10 & Win11 VM's running in VMware (esxi). There are so many annoying issues with the windows RDP client that we are looking to use something else, and FreeRDP looks to be a good alternative if I can get it working.
The classroom setups vary between: 2x screens next to each other, 3x screens where there is an additional screen above the two others, and 3x screens where there is one screen at the bottom, and then two next to each other above the bottom screen.

I've started testing to see if that will work using wfreerdp but I'm struggling with this. +multimon works well and i get a picture on all monitors, but in many cases we only want to use RDP on two or one of the screens. Using for example /monitors:1,2 (or other id arrangements) does not work at all.
Looking at /log-level:trace I can see [gcc_write_client_monitor_data]: MonitorCount=0. I don't know if that's relevant though.

Note that I'm using the static builds from: https://ci.freerdp.com/job/freerdp-nightly-windows/arch=win64,label=vs2017/
The nightly builds from https://cloudbase.it/freerdp-for-windows-nightly-builds/ give the same results, but with much less log info for some reason.

**Comment by akallabeth** [comment, marked as answer]
`wfreerdp` only supports `fullscreen/multimon` on one or all monitors

**Reply by espenu** [reply]
Ah, well, that would explain it then. Thanks
Are there other clients for windows that can be recommended for this?
